### ISSUE TITLE
Generate pdf on fly

### ISSUE THREAD

**Issue by cissemy** [rdvojmoc/DinkToPdf#140]
Hi,
is it possible to create the pdft on fly (in memory) isntead of using  Out = @"C:\DinkToPdf\src\DinkToPdf.TestThreadSafe\test.pdf"?
Thanks

**Comment by marexv** [comment]
Yes. You can return a ByteArray. Dont specifiy the "Out" properti in ObjectSettings settings and nothing will be saved.
